src/panels/config/backup/components/config/ha-backup-config-agents.ts

```diff
@@ -57,26 +57,54 @@ class HaBackupConfigAgents extends LitElement {
       );
     }
 
+    const texts: (TemplateResult | string)[] = [];
+
     const encryptionTurnedOff =
       this.agentsConfig?.[agentId]?.protected === false;
 
     if (encryptionTurnedOff) {
-      return html`
+      texts.push(html`
         <span class="dot warning"></span>
         <span>
           ${this.hass.localize(
             "ui.panel.config.backup.agents.encryption_turned_off"
           )}
         </span>
-      `;
+      `);
     }
 
     if (isNetworkMountAgent(agentId)) {
-      return this.hass.localize(
-        "ui.panel.config.backup.agents.network_mount_agent_description"
+      texts.push(
+        this.hass.localize(
+          "ui.panel.config.backup.agents.network_mount_agent_description"
+        )
       );
     }
-    return "";
+
+    const retention = this.agentsConfig?.[agentId]?.retention;
+
+    if (retention) {
+      if (retention.copies === null && retention.days === null) {
+        texts.push(
+          this.hass.localize("ui.panel.config.backup.agents.retention_all")
+        );
+      } else {
+        texts.push(
+          this.hass.localize(
+            `ui.panel.config.backup.agents.retention_${retention.copies ? "backups" : "days"}`,
+            {
+              count: retention.copies || retention.days,
+            }
+          )
+        );
+      }
+    }
+    return html`${texts.map(
+      (text, index) =>
+        html`${text}${index < texts.length - 1
+          ? html`<span> ⸱ </span>`
+          : nothing}`
+    )}`;
```

Comment:
retur
```suggestion
    return join(values, html`<span> ⸱ </span>`);
```

Reply:
No this doesn't work. You cannot add a `Template` to a `Array.join` moreover it would break the `encryptionTurnedOff` text because it returns `[Object, Object]`